### DISCUSSION TITLE
Citadel Of Algeroth (ALGEROTH) [R]

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#8] [Card Balance]
Citadel Of Algeroth (ALGEROTH) [R]
INFLUENCE, RESTRICTED
At the beginning of your turn generate a random Equipment and attach it to Citadel of Algeroth. It gains ALGEROTH affiliation and is a TOKEN. Will never hold more than 4 cards. Dark Legion Warriors in your SQUAD may take Equipment from the Citadel as one action each

**Comment by sjberry82** [comment]
I was thinking we should cut down on the randomness. The equipment pool could be dependent on the opponent's Discard pile. That way you aren't generating OP equipment cards the opponent has no chance against. Citadel becomes more powerful as the game progresses. Works well with Equipment hate cards.

"At the beginning of your turn generate an Equipment card from Opponent's Discard Pile."

We could also do something with the Citadel when it is full. Like discarding the Equipment CoA creates for an effect.

**Reply by TomekPL-1** [reply]
randomness is fun! If you design card how I proposed then Citadel of Algeroth will be as strong as similar cards from other factions.

**Comment by Kite80** [comment]
Thinking about lore, figure out what happens inside a citadel of Algeroth: in simple words it is a huge factory of monstruous soldiers and corrupted weapons. Its production is very fast, as long there are "raw resources" available. I think reducing such production to equipment physically available in opponent's discard pile is a big limitation, even if that way production doesn't cost anything to the player. A better option for me is to reduce the action cost to 1 and to add a DP cost based on which card the citadel would produce, let's say 2DP for a common card, 4DP for UC, 7DP for R and 10DP for UR. Like this:

 Citadel Of Algeroth (ALGEROTH) [R]
INFLUENCE, RESTRICTED
1 Action: Choose the rarity of the random equipment card the Citadel will generate - 3DP for C, 5DP for UC, 7DP for R or 10DP for UR. It gains ALGEROTH affiliation.

**Comment by 21-ric** [comment]
Citadel Of Algeroth (ALGEROTH) [R]
INFLUENCE, RESTRICTED
WEAPON in opponent's discard pile are attached to this card. 
As 1 action you may attach a WEAPON from this card. Ignore FACTION restriction.
If discarded, attached WEAPON are ANNIHILATED.

**Comment by TomekPL-1** [comment]
My idea was to merge current ability of Citadel of Algeroth with this card
![armory](https://user-images.githubusercontent.com/83038538/116030045-fe617780-a65a-11eb-9124-04906d6b8ebe.jpg)

**Comment by Kite80** [comment]
Anyway, Citadel of Algeroth should not produce UNIQUE, non-DL equipments, like corporate and brotherhood relics
